.gitattributes

```diff
@@ -0,0 +1,5 @@
+uto detect text files and perform LF normalization
```

Comment:
I'm pretty sure this is a typo and this was meant to begin with a '#' to be a comment line.